packages/salesforcedx-vscode-apex/package.json

```diff
@@ -68,7 +67,7 @@
   "scripts": {
     "bundle:extension": "npm run bundle:extension:build && npm run bundle:extension:copy",
     "bundle:extension:copy": "cp ./out/apex-jorje-lsp.jar ./dist/",
-    "bundle:extension:build": "esbuild ./src/index.ts  --bundle --outfile=dist/index.js --format=cjs --platform=node --external:vscode  --external:@salesforce/core --external:applicationinsights --external:shelljs --external:@salesforce/source-deploy-retrieve --external:@salesforce/source-tracking --minify",
```

Comment:
This is worth a shot, but may still require SDR to be not bundled b/c this extension depends on `salesforcedx-utils-vscode` -> depend on SDR

Reply:
I ran E2E with this branch's vsix w/o marking sdr as external and tests passed. I also tried w.o source tracking, but that crashed and burned.